blender/LilySurfaceScraper/metadataHandler.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2021 Zivoy
```

Comment:
I'd rather call this file simply `Metadata.py` since all it does is to define the `Metadata` class (and there seems to be no notion of "handler" involved here)

Reply:
I called it metadataHandler because it makes it more convenient to work with the metadata, loading and saving it